client/src/components/Query/Statement.jsx

```diff
@@ -327,6 +328,25 @@ class Statement extends React.Component {
     }
   }
 
+   handleNewQuery() {
+     const { draft , display } = this.state;
+
+     const key = uuidv1();
+     const instructions = cloneDeep(DEFAULT_INSTRUCTIONS)[0]
+     const draftQuery = {   "instructions" : [] ,
+                            "key": key}
+     draft.push(draftQuery)
+
+     const displayQuery = {"compoundOperators" : true}
```

Comment:
@claudia1296 "Default" display options should come from the parent configuration.